register.js

```diff
@@ -0,0 +1,26 @@
+const register = require("@swc/register/lib/node").default;
+
+register({
```

Comment:
Will we need to allow users to specify any of these configuration properties?

Reply:
Yes, but breaking this into a separate file and allowing the config to be overwritten seems to be causing the error I was running into yesterday. As it stands, users can still overwrite the config by including a `.swcrc`.

Reply:
My inclination is to get the critical path in place first to unblock the getting started section of the website and then come back and figure out the problem with making this composable. Doing so is a non-breaking change with the hook in place so I don't think it makes sense to block finishing the getting started guide.